packages/jest-util/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const _ = require('lodash');
```

Comment:
Let's remove lodash until we actually have a really significant reason to use it within Jest.